conductor/lib/maya_utils.py

```diff
@@ -516,42 +526,55 @@ def collect_dependencies(node_attrs):
                     # directory (i.e. it doesn't have any real smarts about path resolution, etc).
                     # NOTE: that this command will oftentimes return filepaths with an ending "/" on
                     # it for some reason. Strip this out at the end of the function
-                    path = cmds.file(plug_value, expandName=True, query=True, withoutCopyNumber=True)
+                    path = os.path.normpath(cmds.file(plug_value, expandName=True, query=True, withoutCopyNumber=True))
```

Comment:
Same answer as above...

Reply:
In general, I've tried to narrow the places in our client code for where/when paths are manipulated/conditioned.  The majority of the time it occurs _after_ dependency scraping (rather than before, or during).  Specifically the [`process_upload_filepath`](https://github.com/AtomicConductor/conductor_client/blob/master/conductor/lib/file_utils.py#L119) function is called to explode/resolve/normalize any paths that come out of the dependency scraping phase.  See [here](https://github.com/AtomicConductor/conductor_client/blob/master/conductor/lib/conductor_submit.py#L535). 
There certainly are exceptions to this (`process_upload_filepath` gets called several times within the `collect_dependencies` function), but they occur within isolated logic leaves (not affecting other logic leaves).  
I'm not saying that this  _will_ break anything.  But I wouldn't be surprised if it did.

Reply:
It's needed hear so that it can be compared to values in `exlude_paths` and `leaf_paths`